pyomo/core/kernel/expr_coopr3.py

```diff
@@ -638,7 +638,11 @@ def to_string(self, ostream=None, verbose=None, precedence=0):
             ostream.write("( ")
         first = True
         if self._coef != 1:
+            if self._coef < 0:
+                ostream.write("(")
             ostream.write(str(self._coef))
+            if self._coef < 0:
+                ostream.write(")")
```

Comment:
There was a possibility that two negatives could be written out in a row. GAMS doesn't like that.